pyproject.toml

```diff
@@ -13,7 +13,7 @@ packages = [
 exclude = ["hpcflow/data/demo_data"]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.13"
+python = ">=3.9,<3.13"
 zarr = [
     {version = "^2.16.1", python = "<3.9"},
```

Comment:
I think Zarr can now be set to `2.17.2` for all Python versions, since the `python<3.9` condition will no longer be met.

Reply:
Missed that. The perils of searching for `3.8`...